db/migrate/20160316134241_change_column_in_comment.rb

```diff
@@ -0,0 +1,4 @@
+class ChangeColumnInComment < ActiveRecord::Migration
```

Comment:
These migrations do not have any changes in them.  When run, they won't change the database schema.